<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowser</class>
 <widget class="QDialog" name="FileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Mode</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose a .csv file of the above format:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Browse">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>WaitCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File format: scrambled X,scarmbaled Y,(optional)guessX,(optional)guessY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_noGuess_batch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>from file</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Feature_batch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Center of feature (common to all points):</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_longLat_batch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>By a given Long,Lat  (common to all points):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_latLong_batch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Capture_batch">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectLayer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Feature:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectFeatureComboBox_batch">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectLayer_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Layer:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectLayerComboBox_batch">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Batch">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Batch!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileBrowser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileBrowser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
